packages/@orchidui-vue/src/Elements/Sidebar/OCSidebar.stories.js

```diff
@@ -0,0 +1,258 @@
+import Sidebar from "./OCSidebar.vue";
+import Theme from "../../Theme/OCTheme.vue";
+
+export default {
+  component: Sidebar,
+  tags: ["autodocs"],
+};
+
+export const Default = {
+  argTypes: {
+    icon: {
+      control: "select",
+      options: ["", "circle"],
+    },
+  },
+  args: {
+    isExpanded: true,
+    sidebar_menu: [
+      {
+        title: "Payment",
+        menus: [
+          {
+            active: true,
+            name: 'overview',
+            icon: 'grid', // Sidebar icon
+            path: '/', // path
+            label: 'Overview', // Sidebar label
+          },
+          {
+            active: false,
+            name: 'transactions',
+            icon: 'transaction',
+            path: '/transactions',
+            label: 'Transactions'
+          },
+          {
+            active: false,
+            name: 'bank-payouts',
+            icon: 'dollar-paper',
+            path: '/bank-payouts',
+            label: 'Bank Payouts',
+            children: [
+              {
+                active: true,
+                name: 'reports-hit-pay-balance',
+                path: '/balance',
+                label: 'HitPay Balance'
+              },
+              {
+                active: false,
+                name: 'payouts',
+                path: '/payouts',
+                label: 'Payouts'
+              },
+            ]
+          },
+          {
+            active: false,
+            name: 'bank-payouts',
+            icon: 'dollar-paper',
+            path: '/',
```

Comment:
There is a problem when I click on "Overview" it opens me this sub menu. I think it's because they have the same path.

Reply:
yes, only for tester, i will update to correct menu, path should be unique

Reply:
WIP migration all sidebar menu & icons